docs/guide/protocols/septop.rst

```diff
@@ -0,0 +1,131 @@
+This protocol essentially performs two absolute binding free energy calculations simultaneously in opposite directions,
+by (alchemically) inserting one ligand into the binding site, while removing the other ligand at the same time.
+In contrast to the :ref:`RelativeHybridTopologyProtocol <userguide_relative_hybrid_topology_protocol>`, the two ligand topologies are
+completely separate (meaning there is no common core), making atom mapping unnecessary and allowing transformations between chemically diverse ligands.
+
+The relative binding free energy is calculated through a thermodynamic cycle by transforming one ligand into the other ligand
+both in the solvent and in the binding site.
+
+Restraints are required to keep the weakly
+coupled and fully decoupled ligand in the binding site region and thereby reduce the phase
+space that needs to be sampled. In the :class:`SepTopProtocol <.SepTopProtocol>`
+we apply orientational, or Boresch-style, restraints, as described below.
+
+In this cycle, the interactions of one molecule are turned off while simultaneously turning on interactions of the other molecule both in the solvent and complex phases.
+The relative binding free energy is then obtained via summation of free energy differences along the thermodynamic cycle.
+
+.. figure:: img/septop_cycle.png
+   :scale: 50%
+
+   Thermodynamic cycle for the SepTop free energy protocol.
+
+Scientific Details
+------------------
+
+Orientational restraints
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Orientational, or Boresch-style, restraints are automaticallly (unless manually specified) applied between three protein and three ligand atoms using one bond,
+two angle, and three dihedral restraints. Reference atoms are picked based on different criteria, such as the root mean squared
+fluctuation of the atoms in a short MD simulation, the secondary structure of the protein, and the distance between atoms, based on heuristics from Baumann et al. [2]_.
+
+Partial annihilation scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the :class:`SepTopProtocol <.SepTopProtocol>` the coulombic interactions of the molecules are fully turned off (annihilated) in the respective non-interacting end states.
+The Lennard-Jones interactions are instead decoupled, meaning the intermolecular interactions turned off, keeping the intramolecular Lennard-Jones interactions.
+
+The lambda schedule
+~~~~~~~~~~~~~~~~~~~
+
+Molecular interactions are modified along an alchemical path using a discrete set of lambda windows.
+For the transformation of ligand A to ligand B in the binding site, the following steps are carried out, starting with ligand A being fully interacting in the binding site while ligand B is decoupled.
+
+1. Insert the non-interacting dummy ligand B into the binding site and restrain it using orientational restraints. The contribution of the restraints is calculated analytically.
```

Comment:
Separate issue raised: the description of the lambda schedule is non symmetric, i.e. number 5 describes transferring the ligand A to solvent, but none of these refer to transferring ligand B from the solvent (I think it's what is meant by "insert the non-interacting dummy", but the language probably needs to be improved).